apps/aecore/src/aec_peer_messages.erl

```diff
@@ -26,11 +29,11 @@
 
 serialize_response(Type, {ok, Object}) ->
     SerObj = serialize(Type, Object),
-    serialize(response, #{ result => <<"ok">>,
+    serialize(response, #{ result => true,
```

Comment:
I see this is adaptation according to the [protocol spec](https://github.com/aeternity/protocol/blob/cfac4d31691f1358c00ce28dfb93ea590fcd7bd9/sync/p2p_messages.md#msg_p2p_response). Great!